tests/helpers.ts

```diff
@@ -1,11 +1,36 @@
-import { test as base } from '@playwright/test';
+import { Page, test as base } from '@playwright/test';
 import AxeBuilder from '@axe-core/playwright';
 
-interface AxeFixture {
+// TODO: Use type from core
```

Comment:
Do you think it would make sense to put these E2E tests in the client package so we can use the client types? That way we could use the generated services for CRUD operations (https://github.com/awslabs/iot-application/blob/main/apps/client/src/services/generated/services/DashboardsService.ts)

Reply:
Indeed, that's probably the best way to go. Love the idea. Get the types and the generated services. Additionally, I think we can probably avoid manually setting the bearer token. One issue might be starting the server, since Playwright starts the server using an npm script during a CI run. Probably okay though? I will investigate in a follow up! :)

Reply:
Cool, yeah hopefully the CI setup isn't too bad, thanks!